### ISSUE TITLE
Create Channel (Re)design

### ISSUE THREAD

**Issue by liamcardenas** [lbryio/lbry-desktop#859]
This page does not yet exist in the app, so its not yet a redesign (its just a "design"), we will probably detach channel creation from publish https://github.com/lbryio/lbry-app/issues/851

**Comment by tzarebczan** [comment]
Can this also include the ability to update the channel bid? Currently, if you create a new channel with the same name as a previous channel, the channel is updated with the new bid amount. This isn't apparent to the user. So we need a way to either add more LBC to a channel claim or allow updates.

**Comment by kauffj** [comment]
I'd like to minimize the amount of new work that gets grouped with the design epic. Simply finishing the redesign is already a substantial enough project that merits independent release.

I understand if it makes sense to revise or improve existing views as part of the redesign, but wholly new components or pages are likely out of scope.

**Comment by seanyesmunt** [comment]
++ on the above. Very interested in adding this, but most likely after phase 1 of the redesign is released

**Comment by seanyesmunt** [comment]
closing in favor of #561
